receiver/datadogreceiver/README.md

```diff
@@ -0,0 +1,29 @@
+# Datadog APM Receiver
+
+## Overview
+The Datadog APM Receiver accepts traces in the Datadog Trace Agent Format
+
+###Supported Datadog APIs
+- v0.3 (msgpack and json)
+- v0.4 (msgpack and json)
+- v0.5 (msgpack custom format)
+## Configuration
+
+Example:
+
+```yaml
+receivers:
+  datadog:
+    endpoint: 0.0.0.0:8126
+    read_timeout: 60s
+```
+
+### endpoint (Optional)
+The address and port on which this receiver listens for traces on
+
+Default: `0.0.0.0:8126`
```

Comment:
I believe the recommended practice is to avoid  binding to `0.0.0.0` as per the runtime message:

> Using the 0.0.0.0 address exposes this server to every network interface, which may facilitate Denial of Service attacks","documentation":"https://github.com/open-telemetry/opentelemetry-collector/blob/main/docs/security-best-practices.md#safeguards-against-denial-of-service-attacks

Can it be updated to use `localhost` instead?